### PR TITLE
add link to getakka.net

### PR DESCRIPTION
**Pull request by rkuhn**

### CODE REVIEW

File: _includes/frontpage.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 		<div class="row rel">
 			<div class="span4">
 				<div class="home-box">Build powerful concurrent & distributed applications<br/><span class="bold">more easily</span>.</div>
-				<div class="home-small-box">Akka is a toolkit and runtime<br/> for building highly concurrent,<br/> distributed, and resilient<br/> message-driven applications<br/> on the JVM.</div>
+        <div class="home-small-box">Akka is a toolkit and runtime<br/> for building highly concurrent,<br/> distributed, and resilient<br/> message-driven applications<br/> on the JVM.<br/><br/>The power of Akka is now available on the .NET Framework<br/>and Mono via the <a href="http://getakka.net">Akka.NET project</a>.</div>
```
Comment (drewhk) [inline _includes/frontpage.html]: maybe "project" -> "community project"?

Comment (ktoso) [inline _includes/frontpage.html]: maybe "now" -> "also"? Now sounds good for an announcement ("*now* it's available, it's new, yay!"), but this page will stay like this for a while I guess.

**Comment by ktoso** [comment]
LGTM in general

File: _includes/frontpage.html (hunk 1 above)
Comment (patriknw) [inline _includes/frontpage.html]: +1

**Comment by patriknw** [comment]
LGTM

File: _includes/frontpage.html (hunk 1 above)
Comment (rkuhn) [inline _includes/frontpage.html]: Wrt. “community” I don’t see the need to further qualify: akka.net is a project like akka, with some commercial interest behind it, so I’d leave this as is.

Concerning the “now” I agree that “also” is better, will change.

**Comment by rkuhn** [comment]
now also with news item

File: _includes/frontpage.html (hunk 1 above)
Comment (drewhk) [inline _includes/frontpage.html]: Yes, but it is not backed by us. Any other way to make this clear?

Comment (rkuhn) [inline _includes/frontpage.html]: I see what you mean; let’s wait until we know that this concern is an actual problem, and by that time we’ll hopefully have a new website design with more space for clarifying statements.

**Comment by ktoso** [comment]
News item LGTM

**Comment by rasummer** [comment]
Homepage Text & Link LGTM

**Pull request merged**